packages/lesmis/src/views/AdminPanel/routes/surveys/getSurveysPageConfigs.js

```diff
@@ -2,15 +2,15 @@
  * Tupaia Admin
  * Copyright (c) 2022 Beyond Essential Systems Pty Ltd
  */
+import { surveys } from '@tupaia/admin-panel';
+import { getBooleanSelectFilter } from '../../../../views/AdminPanel/table/columnTypes/getBooleanSelectFilter';
+import { getColumnFilter } from '../../../../views/AdminPanel/table/columnTypes/getColumnFilter';
```

Comment:
Would these benefit from a `columnTypes/index.js`? Not sure how many column types there are; ignore as you see fit